apps/meteor/client/views/account/security/TwoFactorEmail.tsx

```diff
@@ -1,15 +1,18 @@
 import { Box, Button, Margins } from '@rocket.chat/fuselage';
-import { useUser, useTranslation } from '@rocket.chat/ui-contexts';
-import type { ComponentProps, ReactElement } from 'react';
+import { useUser, useTranslation, useSetting } from '@rocket.chat/ui-contexts';
+import type { ComponentProps } from 'react';
 import React, { useCallback } from 'react';
 
 import { useEndpointAction } from '../../../hooks/useEndpointAction';
 
-const TwoFactorEmail = (props: ComponentProps<typeof Box>): ReactElement => {
+const TwoFactorEmail = (props: ComponentProps<typeof Box>): JSX.Element | null => {
 	const t = useTranslation();
 	const user = useUser();
 
+	const disableEmail2FAForOAuth = useSetting('Accounts_TwoFactorAuthentication_email_available_for_oAuth_users');
```

Comment:
This being true, is it not enabled instead of disabled?

Reply:
I had coded this earlier, later we decided to change the setting name and default value, forgot to update the variable name accordingly.